hazelcast/src/main/java/com/hazelcast/client/impl/protocol/ClientExceptionFactory.java

```diff
@@ -323,6 +325,7 @@ public ClientExceptionFactory(boolean jcacheAvailable, ClassLoader classLoader)
         register(NO_SUCH_FIELD_ERROR, NoSuchFieldError.class, ((message, cause) -> new NoSuchFieldError(message)));
         register(NO_SUCH_FIELD_EXCEPTION, NoSuchFieldException.class, ((message, cause) -> new NoSuchFieldException(message)));
         register(NO_CLASS_DEF_FOUND_ERROR, NoClassDefFoundError.class, ((message, cause) -> new NoClassDefFoundError(message)));
+        register(TASKLET_EXECUTION_ERROR, TaskletExecutionException.class, ((message, cause) -> new TaskletExecutionException(message)));
```

Comment:
I am not sure we should do this at all, ideally the TaskletExecutionException should be wrapped in JetException on the server side.

Reply:
Well, we can do something like this we would have JetException but silenced in other places:
```
            Exception silencingException = new TaskletExecutionException("Exception in " + t.tasklet + ": " + e, e);
            t.executionTracker.exception(new JetException("Exception in " + t.tasklet + ": " + e, silencingException));
```